README.md

```diff
@@ -108,41 +247,14 @@ test('Some test case', async function (assert) {
 
   await visit('/');
   await a11yAudit('.modal', axeOptions);
-  assert.ok(true, 'no a11y errors found!');
-});
-```
-
-#### Integration and Unit Tests
-
-The helper is also able to be used Integration/Unit tests like so:
-
-```javascript
-import a11yAudit from 'ember-a11y-testing/test-support/audit';
-
-// ...elided for brevity
-
-test('Some test case', function (assert) {
-  this.render(hbs`{{some-component}}`);
 
-  let axeOptions = {
-    rules: {
-      'button-name': {
-        enabled: false,
-      },
-    },
-  };
-  return a11yAudit(this.element, axeOptions).then(() => {
-    assert.ok(true, 'no a11y errors found!');
-  });
+  assert.ok(true, 'no a11y errors found!');
 });
 ```
 
-As you can see, the usage for all types of tests is pretty much the same. The
-only real difference is Acceptance tests get automatic async handling.
-
-#### Optionally Running a11yAudit
+### Optionally Running audits
 
-Ember A11y Testing also allows you to run audits only if `enableA11yAudit`
+`ember-a11y-testing` also allows you to run audits only if `enableA11yAudit`
```

Comment:
Hmm, I don't think this is quite true? We don't only run audits if the QP is set, we **always** run audits if the QP is set.

Reply:
Fixed up the verbiage.